src/interface/api-template.yaml

```diff
@@ -8,405 +8,17 @@ info:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
-        503:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
-    put:
-      description: |
-        The HTTP request `PUT /consents/{ID}` is used to update a specified consent object.
-        The `{ID}` in the URI should contain the `{ID}` that was used in the `POST /consents`.
-
-        - Called by a `auth-service` to add the credential details
-
-        - Called by a `PISP` to add a signature of the challenge
-      summary: UpdateConsent
-      tags:
-        - consent
-      operationId: UpdateConsent
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '../../node_modules/@mojaloop/api-snippets/thirdparty/openapi3/schemas/ConsentIDPutRequest.yaml'
-              example:
-                requestId: '456'
-                initiatorId: 'dfspa'
-                participantId: 'pispa'
-                scopes:
-                - scope: 'account.balanceInquiry'
-                  accountId: 'dfspa.alice.1234'
-                - scope: 'account.sendTransfer'
-                  accountId: 'dfspa.alice.1234'
-                - scope: 'account.sendTransfer'
-                  accountId: 'dfspa.alice.5678'
-                credential:
-                  id: '5678'
-                  credentialType: 'FIDO'
-                  credentialStatus: 'ACTIVE'
-                  challenge:
-                    payload: 'base64(...)'
-                    signature: 'base64(...)'
-                  payload: 'base64(...)'
-      parameters:
-        #Headers
-        - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/parameters/Content-Type.yaml'
-      responses:
-        202:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/202'
-        400:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/400'
-        401:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/401'
-        403:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/403'
-        404:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/404'
-        405:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/405'
-        406:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/406'
-        501:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
-        503:
-          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
-
+    $ref: '../../node_modules/@mojaloop/api-snippets/thirdparty/openapi3/paths/consents_ID.yaml'
   /consents/{ID}/generateChallenge:
```

Comment:
We can remove this resource. The challenge is now derived by the PISP. See: https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#161-deriving-the-challenge

Reply:
removed generateChallenge endpoint